packages/core/src/payment/strategies/adyenv2/index.ts

```diff
@@ -1,5 +1,3 @@
 export * from './adyenv2';
 
-export { default as AdyenV2ScriptLoader } from './adyenv2-script-loader';
-export { default as AdyenV2PaymentStrategy } from './adyenv2-payment-strategy';
-export { default as AdyenV2PaymentInitializeOptions } from './adyenv2-initialize-options';
+export { default as AdyenV2ScriptLoader } from './adyenv2-script-loader';
```

Comment:
EOL

Reply:
Are we going to move the script loader across as well?

Reply:
From what I remember this will stay due to Google payment strategies rely on this?

Reply:
yes